### PR TITLE
[OB-799] Add Auth scopes definition

### PR DESCRIPTION
**Pull request by nkristianto**
Add Auth scopes list to use by API and consent service.

[OB-799] #done

[OB-799]: https://brankas.atlassian.net/browse/OB-799

### CODE REVIEW

**Comment by nkristianto** [comment]
still not sure to put the scopes under metadata or scopes package.

File: v1/auth/scope.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	// AccountsDelete scope the delete account operation.
+	Scope_AccountsDelete Scope = "https://bank.to/account.delete"
+
+	// CardsCreate scope the create card operation.
+	Scope_CardsCreate Scope = "https://bank.to/card.create"
+
+	// CardsRead scope the read card operation.
+	Scope_CardsRead Scope = "https://bank.to/card.read"
+
+	// CardsUpdate scope theupdate card operation.
```
Comment (Hekewa) [inline v1/auth/scope.go]: theupdate

File: v1/auth/scope.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
+// This is temporary solution to use golang const for authentication scope list.
+// TODO : Find a way to generate it with gunk if possible.
+package auth
+
+const (
+	// TranscationCreate scope the transaction read operation.
```
Comment (Hekewa) [inline v1/auth/scope.go]: "scope for the" update in all comments

File: v1/auth/scope.go [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+	// TransactionRead scope the read operation.
+	Scope_TransactionRead Scope = "https://bank.to/transaction.read"
+
+	// TransactionUpdate scope the update operation.
+	Scope_TransactionUpdate Scope = "https://bank.to/transaction.update"
+
+	// TransactionDelete scope the delete operation.
+	Scope_TransactionDelete Scope = "https://bank.to/transaction.delete"
+
+	// AccountsCreate to perform account operation.
```
Comment (Hekewa) [inline v1/auth/scope.go]: AccountsCreate scope to perform create account operation.

**Review by Hekewa — APPROVED** [review]
Few nits about grammar in the comments

**Pull request merged**